x-pack/plugins/spaces/server/saved_objects/schemas.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+// These fields are not searched or aggregated on
+export const SpacesSavedObjectSchemas = {
+  '8.7.0': schema.object({
+    description: schema.maybe(schema.string()),
+    initials: schema.maybe(schema.string()),
+    color: schema.maybe(schema.string()),
+    disabledFeatures: schema.maybe(schema.string()),
```

Comment:
__nit:__ I think it'd be safe to include `name` as well, schema and mappings aren't mutually exclusive, IIUC.

__issue:__ Hmm, I believe `disabledFeatures` can be required (as it's in `space_schema.ts`) and it should be an array of strings, not just string? I'm really curious why the schema didn't complain though... I actually tried to play with this schema and changed types to incompatible (changed `description` to `schema.number()` and `color` to `schema.object`), but I was still able to create and retrieve Space without any issue, @elastic/kibana-core how is that schema supposed to work?

```ts
    core.savedObjects.registerType({
      name: 'space',
      hidden: true,
      namespaceType: 'agnostic',
      mappings: SpacesSavedObjectMappings,
      schemas: SpacesSavedObjectSchemas,
      migrations: {
        '6.6.0': spaceMigrations.migrateTo660,
      },
    });
```